management/clipper_manager.py

```diff
@@ -555,6 +557,37 @@ def inspect_instance(self):
             s = r.text
         return s
 
+    def set_model_version(model_name, model_version, num_containers=0):
```

Comment:
I added checks to the management frontend's `add_application` and `add_model` methods to reject duplicate adds.